harvard-coventry-university.csl

```diff
@@ -98,7 +97,7 @@
           <label form="short" prefix=" " strip-periods="false"/>
           <substitute>
             <text macro="editor"/>
-            <text macro="anon"/>
+            <text variable="container-title" text-case="title" quotes="true" font-style="normal" text-decoration="none"/>
```

Comment:
given the response you received, you'll want to use `<text variable="title"/>` here, not `container-title`. ditto below.